tutorials/roofit/rf711_lagrangianmorph.py

```diff
@@ -69,45 +69,28 @@
 
 input_hists = {sample: ROOT.TFile.Open(infilename).Get(sample).FindObject(observablename) for sample in samplelist}
 input_datahists = {
-    sample: ROOT.RooDataHist("dh_" + sample, "dh_" + sample, [obsvar], input_hists[sample])
-    for sample in samplelist
+    sample: ROOT.RooDataHist("dh_" + sample, "dh_" + sample, [obsvar], input_hists[sample]) for sample in samplelist
 }
 
 # Plot input templates
 # -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
 
 frame0 = obsvar.frame(Title="Input templates for p_{T}^{V}")
-for sample, color in zip(samplelist, [ROOT.kBlack, ROOT.kRed, ROOT.kBlue]):
+for sample, color in zip(samplelist, "krb"):
     input_datahists[sample].plotOn(frame0, Name=sample, LineColor=color, MarkerColor=color, MarkerSize=1)
 
 # Plot morphed templates for cHq3=0.01,0.25,0.5
 # -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
 
 frame1 = obsvar.frame(Title="Morphed templates for selected values")
-morph_datahist_0p01.plotOn(
-    frame1,
-    Name="morph_dh_cHq3=0.01",
+plot_args = dict(
     DrawOption="C",
-    LineColor=ROOT.kGreen,
-    DataError=getattr(ROOT.RooAbsData, "None"),
-    XErrorSize=0,
-)
-morph_datahist_0p25.plotOn(
-    frame1,
-    Name="morph_dh_cHq3=0.25",
-    DrawOption="C",
-    LineColor=ROOT.kGreen + 1,
-    DataError=getattr(ROOT.RooAbsData, "None"),
-    XErrorSize=0,
-)
-morph_datahist_0p5.plotOn(
-    frame1,
-    Name="morph_dh_cHq3=0.5",
-    DrawOption="C",
-    LineColor=ROOT.kGreen + 2,
-    DataError=getattr(ROOT.RooAbsData, "None"),
+    DataError=None,
     XErrorSize=0,
 )
+morph_datahist_0p01.plotOn(frame1, Name="morph_dh_cHq3=0.01", LineColor="kGreen", **plot_args)
+morph_datahist_0p25.plotOn(frame1, Name="morph_dh_cHq3=0.25", LineColor="kGreen+1", **plot_args)
```

Comment:
Ah I see the test for the `kGreen+1` is in this other commit.

Reply:
Also same comment as for the `None` pythonization: do we want this in Doxygen too to give it even more visibility (and to have it as reference for users).

Reply:
Ok! I added the docs.